### ISSUE TITLE
Create subgraph from existing contract failed on ABI

### ISSUE THREAD

**Issue by tobyjaguar** [graphprotocol/graph-cli#256]
I had an issue creating a subgraph from an existing deployed contract. 

Following this command: 
```
graph init \
  --from-contract <CONTRACT_ADDRESS> \
  --network rinkeby \
  --abi 'path/to/.json' \
  <GITHUB_USER>/<SUBGRAPH_NAME> [<DIRECTORY>]
```
I received the error: `✖ Failed to fetch ABI from Etherscan: Unexpected token C in JSON at position 0`

and quit the process.

I then ran:
```
graph init \
  --from-contract <CONTRACT_ADDRESS> \
  <GITHUB_USER>/<SUBGRAPH_NAME> [<DIRECTORY>]
```
which allowed me to setup the project directory as I answered the prompts successively.

I thought it was the abi file at first, but when I walked through the prompts, the process finished.

**Comment by leodasvacas** [comment]
I could reproduce something similar when trying to use an ABI from a path by following the `graph init` interactive prompt, failing with `TypeError: abi.filter is not a function`.

**Comment by Jannis** [comment]
Fixed by #255.
